eZ/Bundle/EzPublishCoreBundle/Command/NormalizeImagesPathsCommand.php

```diff
@@ -80,13 +92,19 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new SymfonyStyle($input, $output);
+
         $io->title('Normalize image paths');
 
         $io->writeln([
             'Determining the number of images that require path normalization.',
             'It may take some time.',
         ]);
 
+        $this->skipHashing = $input->getOption(self::SKIP_HASHING_COMMAND_PARAMETER);
+        $this->skipHashing
+            ? $io->caution('Images\'s filenames will not be hashed.')
```

Comment:
```suggestion
            ? $io->caution('Images\' filenames will not be hashed.')
```

Reply:
Plural possessive nouns should end up with just `'`.

Reply:
Oopsie, that looks bad indeed 🤦🏻